packages/webdriverio/src/commands/element/refetch.js

```diff
@@ -0,0 +1,23 @@
+export default function refetch () {
```

Comment:
I would not make a new command for this. This needs to happen automatically once an element request fails.

Reply:
I can move it from a command to a webdriverio specific function, but i need it accessible to other areas besides the middleware. Are you alright with me moving it to something like /src/refetch.js and importing it to the specific files?

Reply:
Ok moved to utils; and only invoked inside the middleware.